spec/acceptance/destkeypass_spec.rb

```diff
@@ -1,9 +1,17 @@
 require 'spec_helper_acceptance'
 
-describe 'password protected java private keys', unless: UNSUPPORTED_PLATFORMS.include?(host_inventory['facter']['os']['name']) do
-  # rubocop:disable RSpec/InstanceVariable : Instance variables are inherited and thus cannot be contained within lets
+# rubocop:disable RSpec/InstanceVariable : Instance variables are inherited and thus cannot be contained within lets
+
+describe 'password protected java private keys', unless: UNSUPPORTED_PLATFORMS.include?(os[:family]) do
+  def keystore_command(target, storepass = 'testpass', keypass = 'testkeypass')
```

Comment:
Unfortunately, the function is not the same everywhere. It's different enough in all the places it runs that it can't be easily parameterized in a short, one or two parameter function call. TP has asked if I can instead just stop out the part of the call that builds the path to the keytool and does the `'& '` prepend for Windows and build the rest of the string each time it's called, so I'll attempt that now.